index.php

```diff
@@ -23,6 +22,50 @@
 	+------------------------------------------------------------------------------+
 */
 
+// LANGUAGE
+//
+
+function language($sDefault = 'en')
+{
+	if(!empty($_SERVER['HTTP_ACCEPT_LANGUAGE']))
+	{
+		$aBrowserLanguages = explode(',', $_SERVER['HTTP_ACCEPT_LANGUAGE']);
+
+		foreach($aBrowserLanguages as $sBrowserLanguage)
+		{
+			$sLang = strtolower(substr($sBrowserLanguage, 0, 2));
+
+			if(in_array($sLang, Array('fr', 'en')))
+				return $sLang;
+		}
+	}
+
+	else
+		return $sDefault;
```

Comment:
If `$_SERVER['HTTP_ACCEPT_LANGUAGE']` is not empty but does not contains `en` or `fr`, the function will not return anything. You should remove the `else`, to return the default value if the statement is executed, in all cases.